pkg/process/checks/net.go

```diff
@@ -36,14 +37,33 @@ var (
 
 	// ErrTracerStillNotInitialized signals that the tracer is _still_ not ready, so we shouldn't log additional errors
 	ErrTracerStillNotInitialized = errors.New("remote tracer is still not initialized")
+
+	telemetryTypes = []string{
+		string(network.ConnsBpfMapSize),
+		string(network.ConntrackSamplingPercent),
+		string(network.DNSStatsDropped),
+		string(network.NPMDriverFlowsMissedMaxExceeded),
+	}
+
+	monotonicTelemetryTypes = []string{
```

Comment:
I think it makes more sense to have these defined next to the constants. That way it is harder to forget to update these lists.